pkg/db/postgres.go

```diff
@@ -19,6 +18,29 @@ import (
 	"go.uber.org/zap"
 )
 
+type migrateLogger struct {
+	logger  *zap.SugaredLogger
```

Comment:
Why do we need the migrate logger?

Currently, the verbose option is not used in the code. Can't we use the regular logger?

Reply:
We can use the regular logger, but we cannot get the logs of the steps that migrate executes against the db during the `migrate.Up()` call. I thought we are using the verbose option, that's why I made it related to the `debug` attribute of the connectionFactory. I did this because when troubleshooting the migration the last time, me and Xin have had a hard time knowing which step in the migration failed.  But if we don't need it, I can use the regular logger as before.